ndcube/ndcube_sequence.py

```diff
@@ -231,6 +231,144 @@ def explode_along_axis(self, axis):
         # creating a new sequence with the result_cubes keeping the meta and common axis as axis
         return self._new_instance(result_cubes, common_axis=new_common_axis, meta=self.meta)
 
+    def crop(self, *points, wcses=None):
+        """
+        Crop cubes in sequence to smallest pixel-space bounding box containing the input points.
+
+        Each input point is given as a tuple of high-level world coordinate objects.
+        This method does not crop the sequence axis.  Instead input points
+        are passed to the crop method of cubes in sequence.  Note, therefore,
+        that the input points do not include an entry for world coords only
+        associated with the sequence axis.
+        For a description of how the bounding box is defined, see the docstring
+        of the NDCube.crop method.
```

Comment:
Need to check that `NDCube.crop` is linked correctly on RTD.

Reply:
```suggestion
        of the `ndcube.NDCube.crop` method.
```